fastlane/Fastfile

```diff
@@ -186,6 +187,16 @@ platform :ios do
     
   end
 
+  desc "Run OWASP Dependency Check"
+  lane :check_dependencies do |options|
+    dependency_check_ios_analyzer(
```

Comment:
Is that from this plugin?
https://github.com/alteral/fastlane-plugin-dependency_check_ios_analyzer
Shouldn't we define that dependency somewhere?

Reply:
Unfortunately the dependency had to be added in the repos that use this shared fastfile. See my other PRs.

Reply:
Ok. I thought that maybe there's another way. We have to remember to add that plugin for new projects

Reply:
This gives more background https://github.com/fastlane/fastlane/issues/13762 - also, closed as "too heated" 😮

Reply:
They mentioned `fastlane_require` as a workaround. Maybe we could use that too?
(https://docs.fastlane.tools/advanced/Fastfile/)